### PR TITLE
[PAGOPA-844] pa view

### PR DESCRIPTION
**Pull request by alessio-acitelli**
#### List of Changes

- added pa view

#### Motivation and Context

- divestment lmi

#### How Has This Been Tested?

- manual and junit test

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by alessio-acitelli** [comment]
update_code

**Comment by alessio-acitelli** [comment]
update_code

**Comment by alessio-acitelli** [comment]
update_code

**Comment by alessio-acitelli** [comment]
update_code

**Comment by alessio-acitelli** [comment]
update_code

**Review by FedericoRuzzier — APPROVED** [review]
LGTM 👍

**Review by jacopocarlini — APPROVED** [review]

**Comment by alessio-acitelli** [comment]
update_code

**Comment by alessio-acitelli** [comment]
update_code

**Comment by alessio-acitelli** [comment]
update_code

File: src/main/java/it/gov/pagopa/apiconfig/core/controller/CreditorInstitutionsController.java [hunk 1]
```diff
@@ -738,4 +740,89 @@ public ResponseEntity<Ibans> getCreditorInstitutionsIbans(
...
+              description = "Service unavailable",
+              content =
+              @Content(
+                  mediaType = MediaType.APPLICATION_JSON_VALUE,
+                  schema = @Schema(implementation = ProblemJson.class)))
+      })
+  @GetMapping(
+      value = "/view",
+      produces = {MediaType.APPLICATION_JSON_VALUE})
+  public ResponseEntity<CreditorInstitutionsView> getPaymentServiceProvidersView(
```
Comment (FedericoRuzzier) [inline src/main/java/it/gov/pagopa/apiconfig/core/controller/CreditorInstitutionsController.java:792]: maybe rename getCreditorInstitutionView?

**Pull request merged**